rubocop/cop/migration/large_table_schema_update.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Migration
+      # Checks that migrations updating the schema of large tables,
+      # as defined in the configuration, do so safely. As of writing,
+      # this involves utilizing the `uses_departure!` helper.
+      #
+      # @example
+      #  # good
+      #  class ExampleMigration < ActiveRecord::Migration[6.1]
+      #    uses_departure! if Rails.env.staging? || Rails.env.production?
+      #
+      #    add_column :users, :new_field, :integer, nullable: true
+      #  end
+      #
+      # @example
+      #  # bad
+      #  class ExampleMigration < ActiveRecord::Migration[6.1]
+      #    add_column :users, :new_field, :integer, nullable: true
+      #  end
+      class LargeTableSchemaUpdate < RuboCop::Cop::Base
+        MSG = "Use departure to change the schema of large table `%s`"
+
+        LARGE_TABLES = %i[
```

Comment:
Same note regarding moving this to a config file as for `DENIED_TRANSLATION_KEYS`. For this one especially I can see it being very valuable to be able to link easily to this in the wiki, since the wiki already [mentions departure](https://github.com/otwcode/otwarchive/wiki/Migrations#user-content-Using_ptonlineschemachange).

Moving this to the config file would also make it easier for any other users of the archive software to adjust this cop for their database table sizes.